src/cmap/options.rs

```diff
@@ -85,8 +85,8 @@ pub struct ConnectionPoolOptions {
     /// The declared API version
     ///
     /// The default value is to have no declared API version
-    #[builder(default)]
-    pub server_api: Option<ServerApi>,
+    #[builder(skip, default)]
```

Comment:
The tiniest of nits: can we be consistent about the order when doing both `default` and `skip`, or does it actually matter?

Reply:
I don't believe the order is significant. For the sake of not aborting the evergreen patch in progress, is it alright if I leave it as-is? The `skip` tag will be removed in the future anyways.

Reply:
Sure, I guess so :/